### ISSUE TITLE
Cannot run run_net.py: ImportError: cannot import name 'launch_job'

### ISSUE THREAD

**Issue by ShroukMansour** [facebookresearch/SlowFast#239]
When I run the command 
```
 python tools/run_net.py --cfg configs/Kinetics/SLOWFAST_8x8_R50.yaml \
TRAIN.ENABLE False \
TEST.ENABLE False \
DEMO.ENABLE True \
CHECKPOINT_TYPE pytorch \
DEMO.ENABLE True \
DEMO.DATA_SOURCE video.avi \ 
NUM_GPUS 1 \
NUM_SHARDS  1
```
From the  **SlowFast**  root dir I get error  `ImportError: cannot import name 'launch_job'`

OS: Ubuntu 18
Using the same requirements you provided in INSTALL.md 

@takatosp1  any help please!

**Comment by lequytra** [comment]
Can you confirm that the command `python setup.py build develop` ran successfully? I ran into this error once when forgetting to build :)

**Comment by ShroukMansour** [comment]
@lequytra  Yes I've confirmed it's build and got this msg `Finished processing dependencies for slowfast==1.0`

**Comment by ShroukMansour** [comment]
@I forgot to add the python path `export PYTHONPATH=/path/to/SlowFast/slowfast:$PYTHONPATH` Thanks!

But now I got AssertionError: Torch not compiled with CUDA enabled ERROR!

**Comment by takatosp1** [comment]
Hey @ShroukMansour,

Thanks for playing with PySlowFast!
Just saw your updates and it seems to me that it is a env issue for torch installation? I don't have full context here but it might worth to re-visit your cuda env with PyTorch if possible?

Let us know if you found the issue, keep us posted :)
Thanks,

**Comment by ShroukMansour** [comment]
@takatosp1  I should thank you first for this nice work! 
I don't have a gpu installed so I passed NUM_GPUS 0. I reinstalled pytorch to again using this command `conda install pytorch torchvision cpuonly -c pytorch` but still got the same error

Here is the full command I use to run slowfast `python tools/run_net.py --cfg configs/Kinetics/SLOWFAST_8x8_R50.yaml TRAIN.ENABLE False TEST.ENABLE False DEMO.ENABLE True DEMO.DATA_SOURCE video_path.avi NUM_GPUS 0 NUM_SHARDS 1
`

Thanks,

**Comment by lequytra** [comment]
Hi @ShroukMansour, 

I believe right now we don't have support for CPU yet but it should be coming in a few weeks.

**Comment by ShroukMansour** [comment]
Hi @lequytra, 
ooh okay. I'll be waiting for! 
thanks,

**Comment by bihanikeshav** [comment]
i am aslo getting "AssertionError: Torch not compiled with CUDA enabled ERROR!" when running with NUM_GPU 0, is CPU only not supported?
